pkgs/development/python-modules/pydantic/default.nix

```diff
@@ -25,6 +38,19 @@ buildPythonPackage rec {
 
   nativeBuildInputs = [
     cython
+
+    # dependencies for building documentation
+    flake8
```

Comment:
```suggestion
```
flake8 updates are not going to break that.

Reply:
Please remove it from here https://github.com/samuelcolvin/pydantic/blob/master/Makefile#L113 . We don't want this to fail when we update flake8 and also do not really care if the code is formatted correct or not.